lib/widgets/webviews/webview_full.dart

```diff
@@ -1528,7 +1528,7 @@ class WebViewFullState extends State<WebViewFull> with WidgetsBindingObserver, A
               // We need to inject directly, otherwise these scripts will only load in the next page visit
               for (final script in scriptsToAdd) {
                 await webViewController!.evaluateJavascript(
-                  source: _userScriptsProvider.adaptSource(script.source, _userProvider?.basic?.userApiKey ?? ""),
+                  source: script.source,
```

Comment:
`getCondSources` above already calls `adaptSource` so just removed the call here.